_books/ion-1-1/src/macros/special_forms.md

```diff
@@ -184,10 +184,3 @@ a b c
 
 The `for` special form can only be invoked in the body of template macro. It is not valid to use as an E-Expression.
 
-### `fail`
-
-```ion
-(macro fail (message?) /* Not representable in TDL */)
-```
-
-Causes macro evaluation to immediately halt and causes the Ion reader to raise an error to the user. 
```

Comment:
🗺️ Ion 1.1 only offers branching on expansion cardinality, which means that the only time you could selectively call `fail` would be based on expansion cardinality.

Expansion cardinality can already be enforced by `?`/`!`/`*`/`+`, so we're removing this.